openmdao/docs/feature_reference/building_blocks/solvers/linear/index.rst

```diff
@@ -13,4 +13,5 @@ Linear Solvers
     linear_runonce.rst
     direct_solver.rst
     petsc_ksp.rst
-    scipy_iter_solver.rst
\ No newline at end of file
+    scipy_iter_solver.rst
+    linear_user_defined.rst
```

Comment:
I see you adding the doc to the proper index, but I don't see the new doc in here?

Reply:
I see the doc several commits back.  But it's gone now.  I'm guessing your doc also got removed in the merge conflicts with the new docs struct?

Reply:
Not sure why that didn't get added when I did a git mv. Anyway, I've restored this missing piece.

Reply:
OK, that with the return of the main docs header get you my approval!